index.bs

```diff
@@ -0,0 +1,139 @@
+<pre class='metadata'>
+Title: Audio Session
+Shortname: audio-session
+Level: None
+Status: w3c/ED
+Group: mediawg
+TR: https://www.w3.org/TR/audio-session/
+Repository: w3c/audio-session
+URL: https://w3c.github.io/audio-session/
+Editor: Youenn Fablet, Apple https://www.apple.com/, youenn@apple.com, w3cid 96458
+Editor: Alastor Wu, Mozilla https://www.mozilla.org, alwu@mozilla.com, w3cid 92198
+Abstract: This API defines an API surface for manipulating the bits on
+Markup Shorthands: css no, markdown yes
+</pre>
+<pre class=link-defaults>
+spec:webidl; type:dfn; text:resolve
+</pre>
+
+# Introduction # {#introduction}
+
+People consume a lot of media (audio/video) and the Web is one of the primary means of consuming this type of content.
+However, media on the web does not integrate well with the platform.
+The Audio Session API helps to close the gap with platforms that have audio session/audio focus such as Android and iOS.
+This API will help by improving the audio-mixing of websites with native apps, so they can play on top of each other, or play exclusively.
```

Comment:
I know these content are from the explainer which look good to me for being as an initial spec skeleton. But I would like to raise some points which we could handle/discuss in follow-up commits. 

For my understanding, this API not only affects the interaction with the native apps, it would also help on the audio mixing situations across different tabs, such as handling audio competing for different tabs? If so, should we mention that as well? Also, would this API handle the audio competing inside a tab?

Reply:
These are both great questions we need to answer sooner rather than later.
Can you file an issue?